### DISCUSSION TITLE
Any way to dump config similar to `nextflow config` from inside a pipeline?

### DISCUSSION THREAD

**Discussion by tfenne** [nextflow-io/nextflow#3940] [Q&A]
I'm looking for a way to dump the fully resolved configuration (not just the files, but the resolved/active/layered config contents).  `nextflow config` works well enough, but I want to do this inside a workflow as a final step to store the full active configuration that the workflow was run under.

Is there a mechanism to do this?  I know for the `params` I can just access the `params` builtin and dump the contents of that object, but that misses e.g. process configuration that is applied via config.

**Comment by mribeirodantas** [comment]
I don't think this is possible. I have seen many people asking this question and never manage to find or see a solution to it. I managed to get the dictionary with the config information if it's a single file, but it's not even resolved...
